src/commands/dependabot/automerge.ts

```diff
@@ -59,6 +68,10 @@ export default class AutoMerge extends SfdxCommand {
       char: 's',
       default: false,
     }),
+    squash: flags.boolean({
```

Comment:
if you made this a string with 2 options and default=`merge`, then you could use the flag directly in your `opts` spread below.

And then the orb could have the param have default merge and the same options.  Then, the orb wouldn't need
```
- when:
      condition: <<parameters.squash>>
```
And someday, when someone asks for rebase, we're ok.

Reply:
I've replaced this with `--merge-method` and updated the orb PR to set the default value too, looks better than having to handle multiple conditionals 😄